packages/swapper/src/utils.ts

```diff
@@ -286,7 +286,7 @@ export const checkEvmSwapStatus = async ({
   }
 }
 
-export const getRate = ({
+export const getInputOutputRate = ({
```

Comment:
Renamed to a more declarative version so we don't confuse this with what we call `rates` e.g a `rate quote`